code/game/objects/items/stacks/sheets/leather.dm

```diff
@@ -1,3 +1,10 @@
+#define HIDES_COVERED_FULL 3
+#define PLATES_COVERED_FULL 3
+
+#define DRAKE_HIDES_COVERED_LITTLE 1
+#define DRAKE_HIDES_COVERED_ALMOST 2
```

Comment:
not the biggest fan of this define name, not that big of a deal though

Reply:
well, defines  it as a requirement under the last PR, they told me to make that instead to just 1-2-3 hides. If you dont like name, no problem to change, just say your variant